docs/src/pages/guides/batfish-helpers.md

```diff
@@ -161,6 +161,64 @@ class SideBar extends React.Component {
 ]
 ```
 
+## Split pages
+
+`split-pages` supports groups of pages that uses the [split page pattern](../split-pages/) by generating and collecting `headings`, `products`, `services`, and `platform` arrays for the main page to reference.
+
+### Arguments
+
+- `data`, object. Provided by the data selector.
+
+### Set up in batfish.config.js
+
+```js
+const {
+  buildSplitPages
+} = require('@mapbox/dr-ui/helpers/batfish/split-pages.js');
+
+module.exports = () => {
+  return {
+    dataSelectors: {
+      splitPages: (data) => buildSplitPages(data)
+    }
+  };
+};
+```
+
+### Usage
+
+```js
+import React from 'react';
+import splitPages from '@mapbox/batfish/data/split-pages';
+
+class PageShell extends React.Component {
+  render() {
+    const { children, location, frontMatter } = this.props;
+    const isSplitPage = splitPages[location.pathname];
+
+    return (
+      <PageLayout headings={isSplitPage ? isSplitPage.headings : undefined}>
+        {children}
+      </PageLayout>
+    );
+  }
+}
+```
+
+### Output
+
+- The shape is an object, where the top-level keys are pathnames for top level pages that have split pages.
+  - Each object has a `headings` array. It contains all the headings for the page.
+  - Each object has a `products` array. It contains all the products for the page.
+  - Each object may have a `platforms` array. It contains all the platforms for the page.
+  - Each object may have a `services` array. It contains all the services for the page.
```

Comment:
Since all `products` are assigned one or more `services` should `services` be something that is always present?